src/components/exchange/ExchangeField.tsx

```diff
@@ -1,21 +1,30 @@
-import React, { useCallback, useEffect } from 'react';
-import { TouchableWithoutFeedback, View } from 'react-native';
+import React, {
+  FocusEvent,
+  ForwardRefRenderFunction,
+  MutableRefObject,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { TextInput, TouchableWithoutFeedback, View } from 'react-native';
 import { TokenSelectionButton } from '../buttons';
 import { ChainBadge, CoinIcon, CoinIconSize } from '../coin-icon';
 import { Row, RowWithMargins } from '../layout';
 import { EnDash } from '../text';
 import ExchangeInput from './ExchangeInput';
 import { AssetType } from '@rainbow-me/entities';
+import { Network } from '@rainbow-me/helpers';
 import { useColorForAsset } from '@rainbow-me/hooks';
 import styled from '@rainbow-me/styled-components';
 import { borders } from '@rainbow-me/styles';
+import { ThemeContextProps, useTheme } from '@rainbow-me/theme';
 
 const ExchangeFieldHeight = android ? 64 : 38;
 const ExchangeFieldPadding = android ? 15 : 19;
 
-const CoinIconSkeleton = styled.View({
+const CoinIconSkeleton = styled(View)({
```

Comment:
Can we convert this to the DS?

Reply:
Yeah I did try but was unsure how to handle the background color as there isnt a value that maps to our existing color scale for background colors

Reply:
maybe

```
<AccentColorProvider color={colors.alpha(colors.blueGreyDark, 0.1)}>
  <Box background="accent" style={{...borders.buildCircleAsObject(CoinIconSize)}} />
</AccentColorProvider>
```

this is working for me, not sure if there's a better way to achieve what `buildCircleAsObject` does

Reply:
Yeah thats a nice compromise